src/rail_map.h

```diff
@@ -530,7 +530,25 @@ static inline void MakeRailNormal(Tile t, Owner o, TrackBits b, RailType r)
 	t.m8() = r;
 }
 
+/**
+ * Sets the exit direction of a rail depot.
+ * @param t     Tile of the depot.
+ * @param dir   Direction of the depot exit.
```

Comment:
Param name doesn't match implementation. `dir` is probably better than just `d`.